capstone/src/main/webapp/profile-loader.js

```diff
@@ -50,3 +56,16 @@ function createClubElement(text) {
     + '" formmethod="POST">Leave</button>';
   return liElement;
 }
+
+/** Store edited content from profile page */
+function saveProfileChanges() {
+  const newYear = document.getElementById("edit-year").innerHTML;
+  const newMajor = document.getElementById("edit-major").innerHTML;
+
+  var updateProfile = document.getElementById("update-profile");
+
+  updateProfile.innerHTML = '<input type="hidden" name="new-year" value="' + newYear + '">';
```

Comment:
This is to submit a doPost or doGet to the server, right? Do you think there might be an easier/clearner way to do this? I did a similar thing in an earlier PR, and it looks like something we'll be doing a fair bit. Have you seen any alternatives?

Reply:
Doesn't look like we have found any alternatives yet (from our meeting right now) :(

Reply:
If you haven't already, might be good to see if Sean knows :)